afs/media/js/views/components/reports/project.js

```diff
@@ -14,6 +14,10 @@ define([
             Object.assign(self, reportUtils);
             self.sections = [
                 {'id': 'name', 'title': 'Names and Classifications'},
+                {'id': 'substance', 'title': 'Substance'},
```

Comment:
nit:  I've been removing the single quotes from id/title - can you remove those here?

Reply:
fixed